src/Dfe.PlanTech.AzureFunctions/Functions/ContentfulWebHook.cs

```diff
@@ -7,21 +7,19 @@
 
 namespace Dfe.PlanTech.AzureFunctions
 {
-    public class ContentfulWebHook
+    public class ContentfulWebHook : BaseFunction
     {
-        private readonly ILogger _logger;
         private readonly ServiceBusSender _sender;
 
-        public ContentfulWebHook(ILoggerFactory loggerFactory, IAzureClientFactory<ServiceBusSender> serviceBusSenderFactory)
+        public ContentfulWebHook(ILoggerFactory loggerFactory, IAzureClientFactory<ServiceBusSender> serviceBusSenderFactory) : base(loggerFactory.CreateLogger<ContentfulWebHook>())
         {
-            _logger = loggerFactory.CreateLogger<ContentfulWebHook>();
             _sender = serviceBusSenderFactory.CreateClient("contentful");
         }
 
         [Function("ContentfulWebHook")]
         public async Task<HttpResponseData> WebhookReceiver([HttpTrigger(AuthorizationLevel.Function, "post")] HttpRequestData req)
         {
-            _logger.LogInformation("Received webhook POST.");
+            Logger.LogInformation("Received webhook POST.");

```

Comment:
why the removal of interfacing here?

Reply:
It's not removal of interfacing; I've just moved the old `_logger` to a base class, so I had to expose it with a `protected` field as it was `private` before so the concrete class can't access it otherwise.